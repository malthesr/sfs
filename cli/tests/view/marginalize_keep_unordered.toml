args = [
  "view",
  "-p",
  "0",
  "--marginalize-keep",
  "1,0",
  "tests/view/three_populations.sfs"
]