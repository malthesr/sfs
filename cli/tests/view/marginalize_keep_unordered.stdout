#SHAPE=<3/3>
3 12 21 30 39 48 57 66 75
